### PR TITLE
fix opendkim permissions in docker-compose setup

### PR DESCRIPTION
**Pull request by kimsible**
depends on: https://github.com/wader/postfix-relay/pull/18

### CODE REVIEW

**Comment by Chocobozzz** [comment]
Thanks. Could you add a comment with a link to the postfix-relay PR? So we understand the purpose of this setting

**Comment by kimsible** [comment]
> Thanks. Could you add a comment with a link to the postfix-relay PR? So we understand the purpose of this setting

@Chocobozzz, It's weird, I was pretty sure I put it in the description yesterday with "Rely on : https://github.com/wader/postfix-relay/pull/18"

**Comment by kimsible** [comment]
> Thanks. Could you add a comment with a link to the postfix-relay PR? So we understand the purpose of this setting

The link  to the postfix-relay is in the description

**Comment by Chocobozzz** [comment]
> The link to the postfix-relay is in the description

I mean a comment in the code, below the `OPENDKIM_RequireSafeKeys=no` line

File: support/docker/production/.env [hunk 1]
```diff
@@ -38,6 +38,7 @@ POSTFIX_myhostname=<MY DOMAIN>
 # If you need to generate a list of sub/DOMAIN keys
 # pass them as a whitespace separated string <DOMAIN>=<selector>
 OPENDKIM_DOMAINS=<MY DOMAIN>=peertube
+OPENDKIM_RequireSafeKeys=no
```
Comment (rigelk) [inline support/docker/production/.env:41]:
```suggestion
# see https://github.com/wader/postfix-relay/pull/18
OPENDKIM_RequireSafeKeys=no
```

**Comment by rigelk** [comment]
Thanks @kimsible ; I added the comment, so nothing else to do :slightly_smiling_face:

**Comment by kimsible** [comment]
> Thanks. Could you add a comment with a link to the postfix-relay PR? So we understand the purpose of this setting

Ok, sorry @chocobozzz and thanks @rigelk =)

**Comment by wader** [comment]
Hello, wader/postfix-relay#18 has been merged now. Let me know if there are issues!

**Comment by Chocobozzz** [comment]
Thanks @wader!

**Comment by kimsible** [comment]
The previous issue seems to be fixed with the default docker image =). 
Thanks @wader !

**Pull request merged**